### PR TITLE
WINC-1315: add periodic upgrade from eus 4.16

### PR DESCRIPTION
**Pull request by jrvaldes**
Complements:
- https://github.com/openshift/windows-machine-config-operator/pull/2220

### CODE REVIEW

**Comment by jrvaldes** [comment]
/pj-rehearse  periodic-ci-openshift-windows-machine-config-operator-master-azure-e2e-upgrade-from-eus-4.16

**Review by saifshaikh48 — COMMENTED** [review]
Can we introduce a step + script to reduce code duplication between the eus upgrade and regular upgrade `commands`? This way we maintain 1 location for the 2 upgrade tests

**Comment by jrvaldes** [comment]
> Can we introduce a step + script to reduce code duplication between the eus upgrade and regular upgrade `commands`? This way we maintain 1 location for the 2 upgrade tests

see  https://issues.redhat.com/browse/WINC-1315?focusedId=25517858&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25517858

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -11,6 +11,14 @@ base_images:
     name: "4.17"
     namespace: ocp
     tag: windows-machine-config-operator-bundle
+  previous-eus-bundle:
```
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:14]: Why only 4.16 -> 4.18?
Don't we need to be testing 4.12 -> 4.14 and  4.16 -> 4,18 as well?

Comment (jrvaldes) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:14]: good question; continue in [slack](https://redhat-internal.slack.com/archives/CM4ERHBJS/p1725979869281719?thread_ts=1725885396.355509&cid=CM4ERHBJS)

**Comment by jrvaldes** [comment]
/pj-rehearse ack

pre-submit and periodic job passed.

**Comment by jrvaldes** [comment]
had to run `make jobs` to use build04

@saifshaikh48   could you re-LGTM?

**Pull request merged**